runtime/queries/metricsview_aggregation.go

```diff
@@ -331,6 +343,49 @@ func (q *MetricsViewAggregation) rewriteToMetricsViewQuery(export bool) (*metric
 	return qry, nil
 }
 
+func (q *MetricsViewAggregation) resolveTimestampResult(ctx context.Context, rt *runtime.Runtime, instanceID string) (metricsview.TimestampsResult, error) {
```

Comment:
Can you also add this logic in `runtime/resolvers/metrics.ResolveInteractive`? It's okay to copy-paste since we're deprecating the `runtime/queries` package.